tests/files-upload-access-cache.unit.test.ts

```
@@ -0,0 +1,163 @@
+import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest'
+
+const fileId = 'orgs/test-org/apps/test-app/cache-test.zip'
+const originalCaches = globalThis.caches
+const getPgClientMock = vi.fn()
+const getUserIdFromApikeyMock = vi.fn()
+const getAppByAppIdPgMock = vi.fn()
+const getAppByIdPgMock = vi.fn()
+const checkPermissionPgMock = vi.fn()
+
+function encodeMetadataValue(value: string) {
+  return Buffer.from(value).toString('base64')
```

ESLint

[error] Unexpected use of the global variable 'Buffer'. Use 'require("buffer").Buffer' instead.

(node/prefer-global/buffer)